src/Generator.Tests/GeneratorTest.cs

```diff
@@ -28,6 +28,7 @@ public virtual void Setup(Driver driver)
             options.GeneratorKind = kind;
             options.OutputDir = Path.Combine(GetOutputDirectory(), "gen", name);
             options.Quiet = true;
+            options.GenerateDebugOutput = true;
```

Comment:
Why do we need this for all test projects? Also, where is your actual test?

Reply:
If debug output breaks then tests will fail. Consider it a debug output test. struct/class test is [here](https://github.com/mono/CppSharp/pull/1022/commits/4e735b5b61f7191e2140e39012d5ee517f314574) and delegates issue i cant replicate in a test. Already spent few too many hours on this.. So we will have to do without that test. I just cant spend that kind of time on nice-to-haves for free.

Reply:
@rokups I understand. Consider it done. I'll merge it in the next few days.